### PR TITLE
chore:update manageraccess entity

### PR DESCRIPTION
**Pull request by yvanhenang**
Updated the pom.xml to manage dependencies for access management functionality. Enhanced AccessScope and ManagerAccess domain models to refine permissions and scope definitions. Added integration tests in ManagerAccessRepositoryIT to ensure proper functionality of manager-level access features

### CODE REVIEW

**Review by Koufan-De-King — APPROVED** [review]
Looks good to me!

File: ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/ManagerAccess.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @Enumerated(EnumType.STRING)
+    private AccessStatus status;
+
+    @ElementCollection
+    private Set<TypeOfManagedAccess> managedAccessTypes = new HashSet<>();
+
+    @ElementCollection
+    private Set<AccessScope> scopeOfAccess = new HashSet<>();
+
+}
```
Comment (NkwaTambe) [inline ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/main/java/de/adorsys/ledgers/baam/db/domain/ManagerAccess.java:37]: Good job @yvanhenang 

but to make these clear, i think you should `extend your class to the BankAccountAcess entity`.

let's use `inheritance` as the `BankAccountAcess entity` serves as an entery point to all accesses

File: ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/test/java/de/adorsys/ledgers/baam/db/repository/ManagerAccessRepositoryIT.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    @Autowired
+    private ManagerAccessRepository managerAccessRepository;
+
+    // Variable to store a ManagerAccess object
+    private ManagerAccess managerAccess;
+
+    @BeforeEach
+    public void setup() {
+        // Initialize a ManagerAccess object for testing
+        managerAccess = new ManagerAccess();
```
Comment (NkwaTambe) [inline ledgers-bank-account-access-management/ledgers-bank-account-access-repository/src/test/java/de/adorsys/ledgers/baam/db/repository/ManagerAccessRepositoryIT.java:41]: These is correct but not a good practice for our case..

it should be  `ManagerAcess  managerAccess = new ManagerAccess(); ` and not ` managerAccess = new ManagerAccess();`

**Review by NkwaTambe — CHANGES_REQUESTED** [review]

**Comment by NkwaTambe** [comment]
Hello @yvanhenang  please read again the ticket description for more insight

**Pull request closed without merging**